### ISSUE TITLE
In Statistics/Insights/Best Views Ever, shows only the best date, does not show the number of best views.

### ISSUE THREAD

**Issue by ricjcs** [Automattic/wp-calypso#49279]
### Steps to reproduce the behavior / What actually happened

**1. In wordpress.com interface go to "Statistics", In the top menu click on "Insights"**
2. In "All-time posts, views, and visitors" > "Best Views Ever"; observe that **the date with the best views is shown**.

![Captura de ecrã 2021-01-25 220405](https://user-images.githubusercontent.com/11923491/105772093-93b87700-5f59-11eb-925e-c108e5f09996.jpg)

**4.   Go to "Wp-admin", "Dashboard", Site Stats**
5. In "Best Ever"; observe that it **shows the best number of views ever**. If you click there, it displays the date and other information.

![Captura de ecrã 2021-01-25 220620](https://user-images.githubusercontent.com/11923491/105772124-9f0ba280-5f59-11eb-9e26-de570b945974.jpg)

#### What I expected to happen
In the wordpress.com interface, in addition to the date it should show the number of best views.

**Comment by jeherve** [comment]
Good catch. This would be a nice addition to the Stats view in wp-admin. I'll transfer your issue to the Jetpack repo, where the stats are being developed.
